Signal/src/ViewControllers/ConversationView/Cells/OWSMessageCell.m

```diff
@@ -256,11 +234,9 @@ - (void)updateDateHeader
         self.dateHeaderLabel.hidden = NO;
 
         [self.viewConstraints addObjectsFromArray:@[
-            // Date headers should be visually centered within the conversation view,
-            // so they need to extend outside the cell's boundaries.
-            [self.dateHeaderLabel autoSetDimension:ALDimensionWidth toSize:self.contentWidth],
-            (self.isIncoming ? [self.dateHeaderLabel autoPinEdgeToSuperviewEdge:ALEdgeLeading]
-                             : [self.dateHeaderLabel autoPinEdgeToSuperviewEdge:ALEdgeTrailing]),
+            // TODO: Are data headers symmetric or are they asymmetric? gutters are asymmetric?
```

Comment:
re: this TODO

My understanding is that dates should be horizontally centered between the screen edges (not the the gutters).

Let's try to nip these little questions in the bud by asking the authority. Figuring out the right thing before opening the PR means less throw-away code for me to review. (and then re-review once you get the answer).

Reply:
It's on my list of questions to review with Myles.  I'm trying to batch my questions to him (I already have ~30 from just a day's work).